Cargo.toml

```
@@ -12,59 +12,76 @@ members = [
 ]
 
 [features]
-default=['with-gu-mode', 'with-brass-mode']
-with-gu-mode=['zip', 'awc', 'actix', 'gu-client', 'gu-wasm-env-api', 'futures', 'actix-files','tokio-timer']
-with-brass-mode=['gwasm-api', 'indicatif', 'lazy_static', 'promptly', 'serde_urlencoded', 'serde_with']
+default=['wasi-engine', 'with-lwg']
+#with-gu-mode=['zip', 'awc', 'actix', 'gu-client', 'gu-wasm-env-api', 'futures', 'actix-files','tokio-timer']
+#with-brass-mode=['gwasm-api', 'indicatif', 'lazy_static', 'promptly', 'serde_urlencoded', 'serde_with']
+emscripten-engine=['sp-wasm-engine']
+wasi-engine=[
+    'wasmtime',
+    'wasmtime-wasi',
+    'wasmtime-interface-types',
+    'wasmtime-runtime',
+    'wasi-common'
+]
+with-lwg=['zip', 'awc', 'actix']
 
 [dependencies]
 log="0.4"
 env_logger = "0.6"
-failure="0.1.5"
-sp-wasm-engine = { git = "https://github.com/golemfactory/sp-wasm", branch="wasm-runner" }
+anyhow = "1.0.28"
+sp-wasm-engine = { git = "https://github.com/golemfactory/sp-wasm", branch="wasm-runner", optional=true }
+sha3 = "0.8.2"
+futures="0.3"
+
 structopt = { version="0.2", features = ["wrap_help", "suggestions", "color"] }
 app_dirs = "1.2.1"
 uuid = { version = "0.7", features = ["serde", "v4"] }
 gwasm-dispatcher = { path = "gwasm-dispatcher" }
+humantime = "2.0.0"
 
 serde_json="1.0.40"
 serde= "1.0.99"
 bytes = "0.4"
 
+wasmtime = { version = "0.9", optional=true }
+wasmtime-wasi = { version = "0.9", optional=true }
+wasmtime-interface-types = { version = "0.9.0", optional=true }
+wasmtime-runtime = { version = "0.9.0", optional=true }
+wasi-common = { version = "0.9.0", optional=true }
+ya-client={ version = "0.3", git="https://github.com/golemfactory/ya-client.git" }
```

does it need git dependency?
```suggestion
ya-client = "0.3"
```